pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -911,6 +912,28 @@ func LoadSQLDescsFromBackupsAtTime(
 	return allDescs, lastBackupManifest
 }
 
+// FindLatestReIntroductionByTable finds the endtime of the latest incremental
+// backup that reintroduced each backed up table, as of restore time. Note: this
+// function assumes that each introduced span in the manifest covers a single table.
+func FindLatestReIntroductionByTable(
+	manifests []backuppb.BackupManifest, codec keys.SQLCodec, asOf hlc.Timestamp,
+) (map[descpb.ID]hlc.Timestamp, error) {
+	latestReIntro := make(map[descpb.ID]hlc.Timestamp)
+	for _, b := range manifests {
```

Comment:
I think this method works on the assumption that `manifests` is sorted in increasing EndTime because of how we are unconditionally overwriting the map value? If thats the case can we mention it in the comment.